### ISSUE TITLE
Feature Request: Minimal edits

### ISSUE THREAD

**Issue by matchavez** [HappenApps/Quiver-iOS#59]
Hello,

Since this project seems to be at a standstill, maybe we can get something over the line that would be most helpful. Forgetting anything about cells for the moment, can we get an intermediate step where we can simply edit Plain Text in a note? No more than that; a simple implementation of one JSON line that's written to Dropbox. I can edit later, fancy it up later, but I'm really lacking an ability to jot quick notes. After all, that's the point, right? We can always go back and edit and manipulate in the macOS version later.

Any thoughts on that? Or is Quiver a dead platform?

**Comment by matchavez** [comment]
This might help...
https://github.com/HappenApps/Quiver/issues/1000

**Comment by rdewolff** [comment]
Any news?

**Comment by matchavez** [comment]
Dead as a doornail.

Moved on to Typora/Atom/1Writer, and all markdown. It's a shame; this had the potential to be a million dollar app.

**Comment by rdewolff** [comment]
Sad, indeed, this app has (had?) a great potential!
Any other alternative? I use `iA writer` but am not happy with it. Typora has no iOS version. Never tested 1Writer...

**Comment by rdewolff** [comment]
Well, seems Quiver is coming back very soon : https://twitter.com/happenapps/status/1006259032267673601 👍 💥

**Comment by ylian** [comment]
Yes, I'm back on working on Quiver full-time. Right now working on some urgent fixes on the mac version. Then will seriously update the iOS version.

**Comment by rdewolff** [comment]
Amazng news! Welcome back 🙂

**Comment by tarasis** [comment]
I'm also interested in news. I'm trialing the Mac app at the moment and seriously considering buying it. The lack of write support in the iOS app is slightly off putting.

**Comment by matchavez** [comment]
Full time, but no fix in 15 months. Just get Typora, or use Atom or VSCode with a good markdown plugin.

**Comment by Sidetalker** [comment]
Sad days, hope the dev is doing alright.

**Comment by rdewolff** [comment]
well, there is some activity on the repo in the other threads... just sayin'.

**Comment by Sidetalker** [comment]
@rdewolff link? I don't see anything (but admittedly did not look for long)

**Comment by rdewolff** [comment]
https://github.com/HappenApps/website/commit/f63b1fc048187e5a7de84ea5062d11b4ab899fa9

**Comment by ylian** [comment]
I have been working on three major things in the past year:
1. A major rewrite of the macOS app's UI layer. (model layer was rewritten in 3.2, and the UI rewrite deals with the rest of the codebase).
2. A complete rewrite of the iOS app (converting all code to Swift, use latest practices to implement UI, and enable two-way syncing and editing).
3. I was also writing two editors from scratch: a Markdown editor and a code editor. These replace the old Ace editor which have quite a few unfixable bugs.

All three projects are close to completion. As you can imagine, this has been a lot of work for me and thousands of commits.  I am a single developer, Quiver doesn't charge for updates or subscription, and huge amount of work is needed to bring an app to the next level. Please understand.

**Comment by rdewolff** [comment]
Thanks for the update @ylian - I think people would have loved just a quick heads up to know the status.

Sometimes a shared public roadmap helps :)

**Comment by matchavez** [comment]
And while this may seem snarky, it's not intended. You really should just be doing one of these at a time. If you're the only developer, you especially need to not get overwhelmed.

Best of luck.

**Comment by tarasis** [comment]
Great to hear @ylian thank you for the update. I'll echo @rdewolff's suggestion of a public roadmap somewhere, doesn't need dates, but just a rough list of what your aiming for.
